index.js

```diff
@@ -4,17 +4,33 @@ const child = require('child_process')
 const eos = require('end-of-stream')
 const wrap = require('spawn-wrapper')
 const path = require('path')
+const shell = require('shelljs')
 const jeopardy = path.resolve(__dirname, 'jeopardy.mp3')
 
+const players = {
```

Comment:
Iteration order for objects in JS is stable but should still not be relied on:

https://2ality.com/2015/10/property-traversal-order-es6.html

Instead, this could be an array of `{command, play}` where command is the keys and args is the function that returns the args given the audio path. Makes it a bit clearer that players will be tried in the order they are defined in the array.